front/themes/platinum/style.css

```diff
@@ -0,0 +1,16 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+@tailwind variants;
+
+.theme-platinum {
+  --color-primary: #DDDDDD;
+  --color-secondary: #999CFC;
+  /* ... */
+}
+
+
+.selected {
+  @apply border-r-4  bg-secondary;
```

Comment:
Currently unable to generate css because bg-secondary is giving me errors like 


> The `bg-secondary` class does not exist. If `bg-secondary` is a custom class, make sure it is defined within a `@layer` directive.


This is true for _any_ color, even included colors. This may be a bug in tailwind as other people are having similar issues https://github.com/tailwindlabs/tailwindcss/discussions/6818. I haven't tried older versions of tailwind, yet.